run.sh

```diff
@@ -1,7 +1,8 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I believe this will break unless `bash` is added to the image.

```
$ git diff run.sh 
diff --git a/run.sh b/run.sh
index dcbcdf2..a8833ca 100755
--- a/run.sh
+++ b/run.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 current_time=$(date "+%Y.%m.%d-%H.%M")
 if [[ -z $upload ]]
$ docker run -ti --entrypoint=/bin/sh flan_scan
/ # ls -l /bin/bash
ls: /bin/bash: No such file or directory
/ # exit
```